index.js

```diff
@@ -397,9 +397,14 @@ class Hyperbee extends ReadyResource {
     return new Watcher(this, range, opts)
   }
 
-  _onappend () {
+  getAndWatch (key) {
+    if (!this._watchers) throw new Error('Can only watch the main bee instance')
+    return new EntryWatcher(this, key)
```

Comment:
this function should be async like get so the watcher has node populated to the node or null when its resolved